bot/exts/help_channels/_cog.py

```diff
@@ -393,12 +401,12 @@ async def _unclaim_channel(self, channel: discord.TextChannel, claimant_id: int,
             await _cooldown.remove_cooldown_role(claimant)
 
         await _message.unpin(channel)
-        await _stats.report_complete_session(channel.id, is_auto)
+        await _stats.report_complete_session(channel.id, closed_on)
         await self.move_to_dormant(channel)
 
         # Cancel the task that makes the channel dormant only if called by the close command.
         # In other cases, the task is either already done or not-existent.
-        if not is_auto:
+        if not closed_on:
```

Comment:
this needs to explicitly check for the closed_on reason being `command`
initially i had considered passing False or None for command closings, but since command now passes the string command explicitly, this check will never be true.

```suggestion
        if closed_on == "command":
```